udata/templates/macros/search.html

```diff
@@ -83,7 +83,8 @@
 
 
 {% macro facet_panel(result, name, label, icon=None, url=None) %}
-{% set facet = result.get_facet(name) %}
+{% set facet = result.get_aggregation(name) %}
```

Comment:
I think it might be none for facets/aggs that have been renamed (`organization` vs `user_organization`)